tests/conftest.py

```diff
@@ -446,3 +459,33 @@ def minio_data_connection(
         },
     ) as minio_secret:
         yield minio_secret
+
+
+@pytest.fixture(scope="session")
+def nodes(admin_client: DynamicClient) -> Generator[list[Node], Any, Any]:
+    yield list(Node.get(dyn_client=admin_client))
+
+
+@pytest.fixture(scope="session")
+def junitxml_plugin(
+    request: FixtureRequest, record_testsuite_property: Callable[[str, object], None]
+) -> Callable[[str, object], None] | None:
+    return record_testsuite_property if request.config.pluginmanager.has_plugin("junitxml") else None
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
Can we please add a module scoped one too?

Reply:
why?

Reply:
We update dsc between tests, we should have a sanity check between modules, to ensure the tests are not leaving the clusters unstable.

Reply:
so it should be part of restore

Reply:
between tests -no, but between module runs, it might be a good idea to do it. Restore would fail only if the operation itself failed. Some basic checks are needed between module runs.